requirements.txt

```diff
@@ -1 +1,5 @@
-great-expectations>=0.13.14   # package
+--index-url https://pypi.python.org/simple/
+# Install the target package and its dependencies
+-e .
```

Comment:
I haven't seen `-e .` in a `requirements.txt` file before. Is there a reason why we would want to use this here?

Reply:
It defers the dependencies to `setup.py` and seemed to be an accepted and logical approach for reducing the duplication of dependencies between the two files, especially for the purposes of testing during development.  I'll reference https://caremad.io/posts/2013/07/setup-vs-requirement/ for a better explanation.  I'm in no way tied to this approach :)

Reply:
Thanks for the explanation!